### PR TITLE
banker.az

### PR DESCRIPTION
**Pull request by piquark6046**
### To avoid invalid pull requests, please check and confirm following terms

- [X] This is not an ad/bug report;
- [X] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [X] I have performed a self-review of my own changes;
- [X] My changes do not break web sites, apps and files structure.

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [X] Missed ads or ad leftovers;
- [ ] Website or app doesn't work properly;
- [ ] AdGuard gets detected on a website;
- [ ] Missed analytics or tracker;
- [ ] Social media buttons — share, like, tweet, etc;
- [ ] Annoyances — pop-ups, cookie warnings, etc;
- [X] Filters maintenance.

### Enter the issue address

 - https://github.com/AdguardTeam/AdguardFilters/issues/139027

### Add your comment and screenshots

 - https://github.com/AdguardTeam/AdguardFilters/issues/13766

### Terms

- [X] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

File: BaseFilter/sections/foreign.txt [hunk 1]
```diff
@@ -966,12 +966,10 @@ news.lent.az##.right_rebans_box > object[width="200"][height="250"]
 az##.adsgarden
 az##.adsbyskyline
 ! https://github.com/AdguardTeam/AdguardFilters/issues/13766
+/^https:\/\/banker\.az\/wp-content\/uploads\/[0-9]{4}\/[0-9]{2}\/[0-9]{3}x[0-9]{3}\./
```
Comment (Alex-302) [inline BaseFilter/sections/foreign.txt:969]: Please limit by `$domain` to avoid redundant generic regexp rules.

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: BaseFilter/sections/foreign.txt (hunk 1 above)
Comment (piquark6046) [reply BaseFilter/sections/foreign.txt:969]: The `banker.az` domain is included in the regexp rule.
Is it not safe to proceed?

Comment (Alex-302) [reply BaseFilter/sections/foreign.txt:969]: As I remember, it will be checked on every site.

Comment (piquark6046) [reply BaseFilter/sections/foreign.txt:969]: Did you mean that the domain is also used on another website?
I will change it.

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**